framework/auth/campaigns.py

```diff
@@ -0,0 +1,46 @@
+import httplib as http
```

Comment:
@abought, bearing in mind that this is used only for prereg, a closer look at the 'campaign' code would be great.

Mostly we want to be sure the default case (all existing users') experience is not disrupted.